pkg/artifactory/resource/repository/remote/remote.go

```diff
@@ -364,6 +365,12 @@ var BaseRemoteRepoSchema = map[string]*schema.Schema{
 		StateFunc:        util.FormatCommaSeparatedString,
 		Description:      `The set of mime types that should override the block_mismatching_mime_types setting. Eg: "application/json,application/xml". Default value is empty.`,
 	},
+	"download_direct": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "When set, download requests to this repository will redirect the client to download the artifact directly from the cloud storage provider. Available in Enterprise+ and Edge licenses only.",
```

Comment:
Please append description with "Default value is 'false'.", similar to attribute `list_remote_folder_items`.